cmd/fortifyExecuteScan.go

```diff
@@ -159,13 +176,14 @@ func runFortifyScan(config fortifyExecuteScanOptions, sys fortify.System, comman
 	// Generate report
 	if config.Reporting {
 		resultURL := []byte(fmt.Sprintf("https://fortify.tools.sap/ssc/html/ssc/version/%v/fix/null/", projectVersion.ID))
-		ioutil.WriteFile(fmt.Sprintf("%vtarget/%v-%v.%v", config.ModulePath, *project.Name, *projectVersion.Name, "txt"), resultURL, 0700)
-
-		data, err := generateAndDownloadQGateReport(config, sys, project, projectVersion)
-		if err != nil {
-			return err
-		}
-		ioutil.WriteFile(fmt.Sprintf("%vtarget/%v-%v.%v", config.ModulePath, *project.Name, *projectVersion.Name, config.ReportType), data, 0700)
+		dest := fmt.Sprintf("%vtarget/%v-%v.%v", config.ModulePath, config.ProjectName, config.PullRequestName, "txt")
+		ioutil.WriteFile(dest, resultURL, 0700)
+		log.Entry().Info("Wrote project URL to file: %s", dest)
+		//data, err := generateAndDownloadQGateReport(config, sys, project, projectVersion)
```

Comment:
Cannot accept that reports aren't generated anymore. Documenting compliance to audit standards is a must have and the report must be there to go into Sirius.

Reply:
The reason this is commented out is because the function was entering a long loop and just needed a temporary solution for our pipeline to work... I still need to investigate why this was happening..